src/app/lib/config/services/nepali.js

```diff
@@ -10,7 +10,7 @@ export const service = {
     lang: `ne`,
     articleAuthor: `http://www.facebook.com/bbcnews`,
     articleTimestampPrefix: '',
-    articleTimestampSuffix: 'मा अद्यावधिक',
+    articleTimestampSuffix: ' अद्यावधिक',
```

Comment:
Is the space in front intended?

Reply:
I believe so, otherwise we won't leave a space between the date and the word.

Reply:
Although looking at the test maybe that's wrong...

Reply:
I have removed the space.  Good spot.